platform-tests/src/test/java/org/junit/platform/engine/support/descriptor/FilePositionTests.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.platform.engine.support.descriptor;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * @since 1.0
+ */
+class FilePositionTests{
+
+	
+	void equalsTest() {
+		//test for a return of true when the equals method is passed itself
+		FilePosition file = new FilePosition(0, 0);
+		assertEquals(file, file);
```

Comment:
You also need to test `equals()` by passing in different objects that are _equal_.

Furthermore, you have only tested the "happy path". You should also test for cases where two file position objects are _not equal_.